### PR TITLE
Responsive Figma template 1 project

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
This is a complete responsive of a Figma template 1 project from header to contact form section.

### CODE REVIEW

**Review by Cathella — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🟡

### Hi team 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## The Good Points 👍🏻 

- No linter errors. ✔️
- Used correct Github flow. ✔️
- Good commit history ✔️ 
- Used media queries for the Desktop breakpoint in the CSS file ✔️
- Use both Flexbox and Grid in order to layout the elements on the page ✔️
- Mobile design still looks like expected ✔️
- The hamburger button is gone and Item Navigation is centered ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
👇🏻 This is a future point to consider and I highly recommend that you adopt it.
- Always capitalize your commit messages it makes your repository look professional. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -14,252 +14,400 @@
...
+                </a>
+              </li>
+              <li>
+                <a href=""
+                  ><span class="iconify" data-icon="bx:bxl-medium"></span>
+                </a>
+              </li>
+            </ul>
+          </div>
+          <button class="resume">Get my resume</button>
```
Comment (ShoiraTa) [inline index.html:340]: - [ ] [OPTIONAL]: I strongly recommend you to give more space between small sections:

Yours | Figma
------------- | -------------
image  | image

Right now the footer part of this section looks a bit congested 😢 

- [ ] [OPTIONAL] And please pay attention to difference in button style 😉 
- [ ] [OPTIONAL]: I can see that you updated color of social links on header, however it looks like you forgot to do it for footer 😁 . Please consider setting the color of these links to `#505F79`

File: index.html [hunk 2]
```diff
@@ -14,252 +14,400 @@
...
+              alt="card"
+              class="desktop-images"
+            />
+          </li>
+          <li>
+            <img
+              src="img/Snapshoot Portfolio (2).png"
+              alt="card"
+              class="mobile-images"
+            />
```
Comment (ShoiraTa) [inline index.html:193]: - [ ] As the previous reviewer pointed, the image is still broken,  please consider addressing this issue. 

image

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ismailakinkunmi , I’m Shoira, your reviewer for this PR :computer:,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## Optional suggestions:

_Check the comments under the review._ 

------
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -14,252 +14,400 @@
...
+            placeholder="footer@gmail.com"
+            required
+          />
+          <textarea
+            placeholder="Write your message here"
+            name="user_text"
+            required
+            maxlength="500"
+          ></textarea>
+          <button type="submit">Get It Touch</button>
```
Comment (ahmetbozaci) [inline index.html:406]: ### OPTIONAL

- [ ] Your button is not aligned like in the original. To stick to the design you can fix it.

| Your design | Original |
| ------------- |:-------------:|
| image    | image    |

File: index.html [hunk 4]
```diff
@@ -14,252 +14,400 @@
...
+              </li>
+            </ul>
+          </div>
+          <button class="resume">Get my resume</button>
+        </div>
 
-        <ul class="language-list">
-          <li class="langguage">
+        <div class="language-container">
+          <div class="langguage">
```
Comment (ahmetbozaci) [inline index.html:344]: ### OPTIONAL

- [ ] This section space has not like the original. You can fix it.
- [ ] Also after frameworks has horizontal line. You can fix it.
- [ ] You can fix typo on 'Languguages'

| Yours | Original |
| ------------- |:-------------:|
| image       | image      |

File: index.html [hunk 5]
```diff
@@ -14,252 +14,400 @@
...
+          <p class="daily">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div>
+            <button>html</button>
+            <button>css</button>
+            <button class="javascript">javaScript</button>
+          </div>
+          <button class="see-project">See Project</button>
```
Comment (ahmetbozaci) [inline index.html:115]: ### OPTIONAL

- [ ] `See Project `button should take you to the Live Demo of the project or to the GitHub repository. You can fix it.

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 

### Hello @ismailakinkunmi  ✋🏼 
#### Your project is great. I approve of it. There is nothing else to say other than... it's time to merge it and delete the branch. You can submit this project on your dashboard.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Your commit messages should always start with capital letters and you should write an imperative tense. I see some of them are not like that. I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this for your next projects.
Instead of writing `'Added style' or Created the file`' you should write `Add style'  or 'Create the file'`

**OPTIONAL**
- [ ] You can change them by using `git rebase`. If you want to know how to change ı leave a few resources. 
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___


### Cheers and Happy coding! 💻 👏

aproved

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**